utils/acpica-unix/Makefile

```diff
@@ -44,6 +46,18 @@ endef
 define Build/Configure
 endef
 
+define Build/Prepare
```

Comment:
Instead of this kind of hack, you can either use:
```
PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-G$(PKG_VERSION)
HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(PKG_NAME)-G$(PKG_VERSION)
```
OR 
```
TAR_OPTIONS+= --strip-components 1
TAR_CMD=$(HOST_TAR) -C $(1) $(TAR_OPTIONS)
```